src/python/dbs/apis/dbsClient.py

```diff
@@ -5,11 +5,24 @@
 from RestClient.ProxyPlugins.Socks5Proxy import Socks5Proxy
 
 import json
+import gzip
 import os
 import socket
 import sys
 import urllib.request, urllib.parse, urllib.error
 
+def compress(body):
+    """
+    Compress data using gzip
+    """
+    return gzip.compress(bytes(body, 'utf-8'))
```

Comment:
If `body` is not a python3 string, this line will fail with:
```
In [8]: gzip.compress(bytes(b'{"key1":"value1"}', 'utf-8'))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-8-cc5163eb85e1> in <module>
----> 1 gzip.compress(bytes(b'{"key1":"value1"}', 'utf-8'))

TypeError: encoding without a string argument
```

Maybe we could change it to something like:
```
if not isinstance(body, bytes):
    return gzip.compress(bytes(body, 'utf-8'))
return gzip.compress(body)
```

Reply:
Alan, I doubt it is necessary since input would be `json.dumps`, but certainly it does not hurt to make such checks. I adjusted code accordingly.